lib/models/device.ts

```diff
@@ -1288,17 +1289,64 @@ const getDeviceModel = function (
 		 * });
 		 *
 		 * @example
+		 * balena.models.device.getSupervisorTargetState(123, 3).then(function(state) {
+		 * 	console.log(state);
+		 * });
+		 *
+		 * @example
 		 * balena.models.device.getSupervisorTargetState('7cf02a6', function(error, state) {
 		 * 	if (error) throw error;
 		 * 	console.log(state);
 		 * });
 		 */
 		getSupervisorTargetState: async (
 			uuidOrId: string | number,
+			version?: number,
```

Comment:
```suggestion
			version: 2 | 3 = 2
```

Arguably though, the correct thing to do is default to latest but I don’t know the implications

Reply:
agreed, but as they are not the same at all, I prefer to play it safe.